test/aaa_profiling/test_memusage.py

```diff
@@ -1176,10 +1176,10 @@ def test_orm_objects_from_query_w_joinedload(self):
 
         def generate():
             objects = s.query(User).options(joinedload(User.addresses)).all()
-            gc_collect()
```

Comment:
OK can we make a separate test for this - the reason that gc_collect() is there is because this test is specific to the objects being returned.    the separate test can return no objects

Reply:
Sure, let me fix this.

Reply:
@zzzeek Any comment on the fix?

Reply:
force pushed, fixed test.

Reply:
I have to look.  it's fine to change that internal format since it is internal.  not sure if we need the None in there.